controller-server/src/main/java/com/yahoo/vespa/hosted/controller/application/DeploymentJobs.java

```diff
@@ -131,7 +131,15 @@ public boolean isDeployableTo(Environment environment, Optional<Change> change)
     public boolean isDeployed(Change change) {
         return status.values().stream()
                 .filter(status -> status.type().isProduction())
-                .allMatch(status -> isSuccessful(status.type(), change));
+                .allMatch(status -> isDeployed(change, status.type()));
+    }
+
+    /** Returns whether job has successfully deployed change */
+    public boolean isDeployed(Change change,  JobType jobType) {
```

Comment:
Changing from isSuccessful to isDeployed seems odd to me.
For test steps it is not whether we deployed we are interested in but whether we completed successfully, and that is what we actually check for. 
(And when we separate out production tests as separate steps (which runs after delay), isDeployed becomes completely wrong).

Reply:
Changed, thanks.